src/components/NostoCampaign.ts

```diff
@@ -19,8 +19,6 @@ export class NostoCampaign extends HTMLElement {
 
   async connectedCallback() {
     assertRequired(this, "placement")
-    this.classList.add("nosto_element")
```

Comment:
shouldn't be marked as a placement, since it might have additional context (product / variant)